.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+Metrics/LineLength:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
If we're being strict, this is generally something we should try to deal with – maybe exclude the violating files for now and we can go through them later. Same with `Metrics/MethodLength`, `Metrics/CyclomaticComplexity`, `Metrics/PerceivedComplexity`, and `Metrics/AbcSize`.